src/streamlink/plugins/youtube.py

```diff
@@ -205,11 +204,16 @@ def _get_stream_info(self, url):
         if not video_id:
             return
 
-        params = {
-            "video_id": video_id,
-            "el": "player_embedded"
-        }
-        res = http.get(API_VIDEO_INFO, params=params, headers=HLS_HEADERS)
+        for el in ("detailpage", "embedded"):
+            params = {
+                "video_id": video_id,
+                "el": el
+            }
+            res = http.get(API_VIDEO_INFO, params=params, headers=HLS_HEADERS)
+            if "status=fail" in res.text:
+                continue
+            break
+
```

Comment:
I know there is no error checking at the moment for the fail status, maybe now is a good time to add a logging message if there is a fail on both? The current feedback for the youtube plugin is pretty minimal :)